documentation/webpack.config.js

```diff
@@ -29,6 +29,11 @@ const litComponentDirectories = [
 
 const openWcConfig = createDefaultConfig({
     input: path.resolve(__dirname, './index.html'),
+    webpackIndexHTMLPlugin: {
+        polyfills: {
+            webcomponents: true, // Needed until Chromium Edge is released
+        },
+    },
```

Comment:
Is it this, or does https://github.com/adobe/spectrum-web-components/blob/master/documentation/webpack.config.js#L15 need to import [createCompatibilityConfig](https://open-wc.org/building/building-webpack.html#browser-support)?

Reply:
This one polyfill is enough to make the docs work in the current version of Edge for me locally.

Reply:
Then I'm good!